### PR TITLE
WT-3860 lint

### PR DESCRIPTION
**Pull request by keithbostic**

### CODE REVIEW

File: src/utilities/util_main.c [hunk 1]
```diff
@@ -41,7 +41,7 @@ usage(void)
 	    "\t" "compact\t  compact an object\n"
 	    "\t" "copyright copyright information\n"
 	    "\t" "create\t  create an object\n"
-	    "\t" "downgrade\t  downgrade an database\n"
+	    "\t" "downgrade downgrade an object\n"
```
Comment (agorrod) [inline src/utilities/util_main.c]: I'm surprised the explicit tab character is removed. Oh I see - downgrade is the longest work, should we just use spaces or a more explicit separator?
Is it possible to downgrade anything other than an entire database?

Comment (keithbostic) [reply src/utilities/util_main.c]: Thank you, @agorrod, you're correct, and I've backed out the "database" to "object" part of the change.

**Comment by michaelcahill** [comment]
autobuild: retest this please

**Comment by michaelcahill** [comment]
autobuild: retest this please

**Pull request merged**